op-guide/docker.md

```diff
@@ -181,16 +203,44 @@ docker run --net ${net} -d --name tidb \
   -L warn
 ```
 
-## Use SQL client
-After you started a TiDB cluster, you can use official mysql client connecting to TiDB for a test immediately.
+## Step 6. Use the official MySQL client
+Once the TiDB cluster is running, you can use the official MySQL client to connect to TiDB for a quick test.
 
 ```bash
 mysql -h 127.0.0.1 -P 4000 -u root -D test
+# Welcome to the MySQL monitor.  Commands end with ; or \g.
+# Your MySQL connection id is 10001
+# Server version: 5.5.31-TiDB-1.0 MySQL Community Server (GPL)
+```
+
+Then run some SQL statments:
+
+```bash
+mysql> CREATE DATABASE mydb;
+Query OK, 0 rows affected (0.01 sec)
+
+mysql> USE mydb;
+Database changed
+
+mysql> CREATE TABLE mytable ( id INT, data VARCHAR(100), dt DATE, PRIMARY KEY (id) );
+Query OK, 0 rows affected (0.01 sec)
+
+mysql> INSERT INTO mytable VALUES (1, 'test data', '2016-08-03');
+Query OK, 1 row affected (0.00 sec)
+
+mysql> SELECT * FROM mytable;
++----+-----------+------------+
+| id | data      | dt         |
++----+-----------+------------+
+|  1 | test data | 2016-08-03 |
++----+-----------+------------+
+1 row in set (0.00 sec)
 ```
 
-## In another way, using `docker-compose`
+## Alternatively, use `docker-compose`
 
-A simple `docker-compose.yml`:
+Note that you don't need to create a docker network first while using `docker-compose`.
+Only a `docker-compose.yml` file is needed as the following.
```

Comment:
Note: If you are using `docker-compose`, you don't need to create a Docker network. The following `docker-compose.yml` file is enough.